lms/djangoapps/courseware/migrations/0014_auto__add_studentmodulehistoryextended.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import dbs
+from south.v2 import SchemaMigration
+from django.db import models
+
+default_db = dbs['default']
+csmh_db = dbs['student_module_history']
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Archive 'StudentModuleHistory'
+        default_db.rename_table('courseware_studentmodulehistory', 'courseware_studentmodulehistoryarchive')
```

Comment:
I just realized that I can update this so as to not rename the table at all (which also minimizes the rollback penalty). I'm going to do that.